README.md

```diff
@@ -100,6 +100,7 @@ There are multiple supported backends, which can be selected at runtime in the e
 | `/ocl/occa`              | OpenCL OCCA kernels                               |
 | `/gpu/cuda/ref`          | Reference pure CUDA kernels                       |
 | `/gpu/cuda/reg`          | Pure CUDA kernels using one thread per element    |
+| `/gpu/cuda/shared        | Optimized pure CUDA kernels using shared memory   |
```

Comment:
Woops, this is missing a trailing `

Can you make a PR to revert this commit and add the `?

Reply:
I'm sorry...